kie-server-parent/kie-server-tests/kie-server-integ-tests-all/src/test/java/org/kie/server/integrationtests/jbpm/cases/CaseRuntimeDataServiceIntegrationTest.java

```diff
@@ -541,6 +553,75 @@ public void testGetCaseInstancesWithData() {
         assertNotNull (instance.getCaseFile());
         assertEquals("first case started", instance.getCaseFile().getData().get("s"));
     }
+
+    @Test
+    public void testFindCaseTasksAssignedAsPotentialOwnerByPassAuth() throws Exception {
+        String caseId = startUserTaskCase(USER_YODA, USER_JOHN);
+        assertNotNull(caseId);
+        assertTrue(caseId.startsWith(CASE_HR_ID_PREFIX));
+
+        assertTaskListAsPotentialOwner(caseId, USER_YODA, USER_JOHN, true);
+        caseClient.destroyCaseInstance(CONTAINER_ID, caseId);
+
+        changeUser(USER_JOHN);
+
+        caseId = startUserTaskCase(USER_JOHN, USER_YODA);
+        assertNotNull(caseId);
+        assertFalse(caseId.isEmpty());
+
+        assertTaskListAsPotentialOwner(caseId, USER_JOHN, USER_YODA, false);
+        caseClient.destroyCaseInstance(CONTAINER_ID, caseId);
+    }
+
+    @Test
+    public void testFindCaseTasksAssignedAsBusinessAdminByPassAuth() throws Exception {
+        CaseFile caseFile = CaseFile.builder()
+                .addUserAssignments(CASE_OWNER_ROLE, USER_YODA)
+                .addUserAssignments(CASE_CONTACT_ROLE, USER_JOHN)
+                .addGroupAssignments(CASE_PARTICIPANT_ROLE, CASE_ADMIN_GROUP)
+                .build();
+        String caseId = startUserTaskCase(caseFile);
+
+        assertNotNull(caseId);
+        assertTrue(caseId.startsWith(CASE_HR_ID_PREFIX));
+        assertTaskListAsBusinessAdmin(caseId, USER_YODA, USER_YODA);
+        caseClient.destroyCaseInstance(CONTAINER_ID, caseId);
+
+        caseId = startUserTaskCase(caseFile);
+        assertNotNull(caseId);
+        assertTrue(caseId.startsWith(CASE_HR_ID_PREFIX));
+
+        assertTaskListAsBusinessAdmin(caseId, USER_YODA, USER_ADMINISTRATOR);
+        changeUser(USER_YODA);
+        caseClient.destroyCaseInstance(CONTAINER_ID, caseId);
+    }
+
+    @Test
+    public void testFindCaseTasksAssignedAsStakeHolderByPassAuth() throws Exception {
+        assertEquals(USER_YODA, configuration.getUserName()); // Check current authenticated user is yoda
```

Comment:
Is this needed? I would expect that yoda is a default user in he beginning of tests.

Reply:
to avoid relying on the test config itself in case it would be changed in future, I just wanted to make it more robust

Reply:
ok